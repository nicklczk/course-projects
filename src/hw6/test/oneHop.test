# Tests path finding for one hop

# Create graph
LoadGraph friends champions.tsv

# find path one hop
FindPath friends Vincent Kyle