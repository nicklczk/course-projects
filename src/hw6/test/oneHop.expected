# Tests path finding for one hop

# Create graph
loaded graph friends

# find path one hop
path from Vincent to Kyle:
Vincent to Candice via Lander
Candice to Kyle via McMahon